src/components/ItemCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import LikeCount from "../assets/like.svg";
+
+function ItemCard({ item }) {
+  return (
+    <div className="itemCard">
+      <img src={item.images[0]} alt={item.name} className="itemCardThumbnail" />
```

Comment:
item 이 없거나, item.images 가 없을 때에 에러가 날 수 있어서 해당 부분 방어코드 넣어주면 좋을 거 같아요! (무조건 오는 값이라면 무시해주세요)
```suggestion
      <img src={item.images?.[0]} alt={item.name} className="itemCardThumbnail" />
```